src/DynamoCore/Core/PathManager.cs

```diff
@@ -355,6 +355,17 @@ internal string GetBackupFilePath(WorkspaceModel workspace)
             return Path.Combine(BackupDirectory, fileName);
         }
 
+        internal void LoadPackageFolders(IEnumerable<string> folders)
+        {
+            // Preserve the priority specified in 'folders' list by first removing
+            // elements in 'packageDirectories' that are also present in 'folders'
+            var remainingFolders = packageDirectories.Where(x => !folders.Contains(x));
+            packageDirectories.Clear();
```

Comment:
Due to lazy evaluation, `remainingFolders` will be empty as soon as `packageDirectories.Clear` is called. So it's always empty moving forward.